pkg/cmd/codespace/rebuild_test.go

```diff
@@ -0,0 +1,36 @@
+package codespace
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cli/cli/v2/internal/codespaces/api"
+	"github.com/cli/cli/v2/pkg/iostreams"
+)
+
+func TestAlreadyRebuildingCodespace(t *testing.T) {
+	app := testingRebuildApp()
+
+	err := app.Rebuild(context.Background(), "rebuildingCodespace")
+	if err != nil {
+		t.Errorf("rebuilding a codespace that was already rebuilding: %v", err)
+	}
+}
+
+func testingRebuildApp() *App {
+	rebuildingCodespace := &api.Codespace{
```

Comment:
This is a bit specific for the single test above. If we added a second test for the normal (non-rebuilding) case then this becomes a hinderance. Maybe instead we could pass in the mock codespace which is constructed in the main test body